integration/tests/cli/general/test_help_command.rs

```diff
@@ -21,6 +21,7 @@ Commands:
   ping             ping iggy server
   me               get current client info
   stats            get iggy server statistics
+  snapshot         snapshot - gathers and packages system diagnostics and logs for troubleshooting and analysis
```

Comment:
IMO this command description does not fit here, please align to other commands - drop second snapshot and make the description like collect iggy server troubleshooting data

Reply:
description was updated as suggested